### PR TITLE
Spinner

### PR DESCRIPTION
**Pull request by Yjohn**
created a spinner while waiting for the server respond

### CODE REVIEW

File: server/routes/admin.js [hunk 1]
```diff
@@ -42,6 +43,15 @@ router.post('/places/add', (req, res) => {
   placesDB.addNewPlace(query, callBack)
 })
 
+router.post('/answers', (req, res) => {
```
Comment (kabaros) [inline server/routes/admin.js]: I don't think this is needed or used anywhere - there is no `add answer` in the Admin panel only in the App (the api)

Comment (Yjohn) [reply server/routes/admin.js]: I am trying to get the answers, after the client post ...
the questioner doesn't work at this moment

Comment (kabaros) [reply server/routes/admin.js]: yeah then then should be in `api.js` - not `admin.js` - the App doesn't know anything about admin

Comment (kabaros) [reply server/routes/admin.js]: the other bits you added to api are correct - but not admin

**Pull request merged**